CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Wayland primary selection clipboard not storing text when selection is stopped outside of the window
 - Block URL highlight while a selection is active
 - Bindings for Alt + F1-F12
+- Transparency when cell background color was equals to named background color
```

Comment:
I feel like this might be a bit tough to understand, especially because named colors are an Alacritty internal detail.

Maybe this works better:

```
- Opacity incorrectly applying to cells with their background color matching the teriminal background
```

Not ideal, but it should be understandable and allow people who ran into this problem before to properly identify that it has been fixed.